app/ui/src/app/support/support.component.scss

```diff
@@ -0,0 +1,72 @@
+::ng-deep .toolbar-pf {
+  box-shadow: none;
+  border: solid #ededed;
+  border-width: 1px 0 0;
+
+  .toolbar-pf-results {
+    border-top: 1px solid #ededed;
+
+    .list-inline {
+      margin: 0;
+    }
+    
+  }
+
+  .form-group {
+    border-right: none;
+  }
+
+  @media only screen and (max-width : 992px) {
+    border-width: 1px 0;
+  }
+
+}
+
+.indent {
+  margin-left: 20px;
+}
+
+.mainform {
+  background-color: white;
+  padding: 20px;
+
+  .integrations {
+    margin-bottom: 10px;
+  
+    .toolbar_row {
+      position: relative;
+
+      .toolbar_helpers {
+        display: flex;
+        align-items: center;
+        position: absolute;
+        right: 20px;
+        top: 13px;
+
+        @media only screen and (max-width : 992px) {
+          position: static;
+          padding-top: 10px;
+        }
+  
+        .selected_items {
+          border-right: 1px solid #d1d1d1;
+          margin-right: 20px;
+          padding-right: 20px;
+        }
+  
+      }
+
+      @media only screen and (max-width : 992px) {
+        display: block;
+        padding-bottom: 10px;
+      }
+  
+    }
+  
+  }
+
+  .download {
+    padding-top: 10px;
+  }
+
+}
```

Comment:
When coding SASS for Angular components (and [Web Components](https://en.wikipedia.org/wiki/Web_Components) in general), you need to bear in mind the following: The compiled CSS will be injected inline upon instantiating the component dynamically, and the composed CSS will be removed from the DOM upon destroying such component. In that sense, the lesser CSS we inject in the component style directives, the better. The rationale for that is because we're ultimately creating a visual language though components, and therefore components are meant to be standard and share the same look and feel. That's not doable when each component features its own visual ruleset. 

On the other hand, refining the styles of each UI element on a per component basis is not scalable. A regular components-driven application can feature literally hundreds (or thousands!) of components. Tweaking CSS one component at a time each time the UI changes is simply not doable. As a rule of thumb, the styles defined on a single component are just meant to _override_ any generic style that affects such component because of its very particular nature or behavior. 

Just a few principles when adding SCSS code to the codebase:

- First and foremost, ask to yourself if that style will be eventually reused by other instances of the application. If so, then consider adding it as part of our general styles available at `app/src/ui/scss`.
- By importing the application-level `@import 'syndesis-sass';` you'll gain immediate access yo our application-level variables, mixins and functions (including our vendor variables such as PF's color palette variables!). The more reusable our CSS code is, the more lean it becomes.
- Avoid hardcoding values in the stylesheets. We have generic gutters defined there, ready for use in margins and paddings. If you feel that none of those suit your requirements, that is a good sign that this component layout breaks the overall rules and then deserves a custom approach, but think twice before goind down that way.
- Following up with this, never hardcode colors in your code. use the PF values instead.
- The SCSS code should allow for getting a general idea of the DOM layout and component hierarchy just by inspecting its indentation and selector names. Please, use the [BEM conventions](http://getbem.com/introduction/) to accomplish this.
- Avoid using generic names such as `.download` or `.thing` and feel free to use custom className prefixes such as `.syn-`. This ensures that our custom styles are not affected by vendor stylesheets shoudl we introduce third party CSS libraries in the future.
- If you create helper classnames that affect a DOM/custom element default behavior, use modifier prefixes to make the helper more self-explanatory, such as `.is-` (for toggable styles) or `.has` for overloaded styles (when not using BEM's `--modifier` suffix pattern)